packages/eslint-config-skuba/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "eslint-config-skuba",
+  "version": "3.1.0",
+  "private": false,
+  "description": "ESLint config for skuba",
+  "homepage": "https://github.com/seek-oss/eslint-config-skuba#readme",
+  "bugs": {
+    "url": "https://github.com/seek-oss/eslint-config-skuba/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/seek-oss/eslint-config-skuba.git"
+  },
+  "license": "MIT",
+  "sideEffects": false,
+  "main": "./index.js",
+  "module": "./index.js",
+  "types": "./index.d.ts",
+  "files": [
+    "index.d.ts",
+    "index.js"
+  ],
+  "scripts": {
+    "format": "skuba format",
+    "lint": "skuba lint"
+  },
+  "dependencies": {
+    "@types/eslint": "^8.4.1",
+    "@typescript-eslint/eslint-plugin": "^6.13.2",
+    "@typescript-eslint/parser": "^6.13.2",
+    "eslint-config-seek": "^12.0.1",
+    "eslint-plugin-jest": "^27.6.0",
+    "eslint-plugin-tsdoc": "^0.2.14",
+    "eslint-plugin-yml": "^1.5.0"
+  },
+  "devDependencies": {
+    "eslint": "^8.11.0",
+    "skuba": "workspace:*",
```

Comment:
I think we may be able to remove this as a dev dep as `format` and `lint` can be run at the root: https://github.com/seek-oss/aws-codedeploy-hooks/blob/c493faa8265345302359cb5bf3aae8770f882b1f/package.json#L8-L9